src/DataCollector/SessionCollector.php

```diff
@@ -30,6 +35,13 @@ public function collect()
         foreach ($this->session->all() as $key => $value) {
             $data[$key] = is_string($value) ? $value : $this->formatVar($value);
         }
+
```

Comment:
Hi @erikn69 
this won't handle deep array values as they already converted to string, probably all collect() should look like 
```
    /**
     * {@inheritdoc}
     */
    public function collect()
    {
        $data = $this->session->all();

        foreach ($this->hiddens as $key) {
            if (Arr::has($data, $key)) {
                Arr::set($data, $key, '******');
            }
        }

        foreach ($data as $key => $value) {
            $data[$key] = is_string($value) ? $value : $this->formatVar($value);
        }


        return $data;
    }

```
i.e. apply hiddens before converting values to strings